src/systems/filecoin_blockchain/struct/chain/chain.go

```diff
@@ -23,7 +23,8 @@ func (chain *Chain_I) TipsetAtEpoch(epoch abi.ChainEpoch) Tipset {
 func (chain *Chain_I) RandomnessSeedAtEpoch(epoch abi.ChainEpoch) abi.RandomnessSeed {
 
 	ts := chain.TipsetAtEpoch(epoch)
-	return ts.MinTicket().Digest()
+	//	return ts.MinTicket().Digest()
```

Comment:
`Digest()` doesn't exist, so I reverted this code to return `DrawRandomness(epoch)` as the version previous to the broken PR.

Reply:
Unless DrawRandomness does the digest (which I can't tell because it's now a reference to actors code), this isn't right and the digest must be taken afterwards.

Reply:
`DrawRandomness` is referencing the local (replaced in go.mod) version of `spec-actors`, which if I understand what you mean should be the `source of truth` for what gets implemented / copied to `github.com/filecoin-project/specs-actors`

If it was referencing the latest version at `github.com/filecoin-project/specs-actors`, I think this won't compile, so I am not sure which is really the correct version we want to use.

Reply:
No, the vendored actors repo is in no way a source of truth. It just points at an old commit of github.com/filecoin-project/specs-actors reflecting the point in time at which we liberated the actors code from the spec.

- Best would be that the relevant algorithm is expressed directly in this spec as pseudocode
- Slightly worse would be as code
- Worst of all is to reference an external repo like specs-actors